tests/tests/fv3_gfdlmprad

```diff
@@ -56,6 +56,7 @@ export LIST_FILES="atmos_4xdaily.tile1.nc \
 export_fv3
 
 export TASKS=192
+export WLCLK=60
```

Comment:
These longer runtimes need to be investigated.

Reply:
A full 60 minutes is not required but this is what I tested out of frustration. The runs ended up taking about 20-25 minutes in my case.